config/locales/client.en.yml

```diff
@@ -3613,8 +3616,8 @@ en:
       filter_to:
         one: "%{count} post in topic"
         other: "%{count} posts in topic"
-      create: "New Topic"
-      create_long: "Create a new Topic"
+      create: "New topic"
+      create_long: "Create a new topic"
       open_draft: "Open Draft"
```

Comment:
There are many more places with titleized labels, I limited myself to the ones agreed.